bundles/org.openhab.binding.roku/README.md

```diff
@@ -123,6 +128,8 @@ String Player_ChannelName        "Channel Name: [%s]"        { channel="roku:rok
 String Player_ProgramTitle       "Program Title: [%s]"       { channel="roku:roku_tv:mytv1:programTitle" }
 String Player_ProgramDescription "Program Description: [%s]" { channel="roku:roku_tv:mytv1:programDescription" }
 String Player_ProgramRating      "Program Rating: [%s]"      { channel="roku:roku_tv:mytv1:programRating" }
+Switch Player_Power              "Power: [%s]"               { channel="roku:roku_tv:mytv1:power" }
+String Player_PowerState         "Power State: [%s]          { channel="roku:roku_tv:mytv1:powerState" }
```

Comment:
There is more than on/off coming from the device.  For example, when off my TVs say "Ready".  They can say different things depending on the low power mode settings.  It could be relevant to some, so I included it.